draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModal.tsx

```diff
@@ -1,23 +1,48 @@
 import * as React from "react"
 import { createPortal } from "react-dom"
 import FocusLock from "react-focus-lock"
+import uuid from "uuid/v4"
 const { CSSTransition } = require("react-transition-group")
+import {
+  ModalAccessibleContext,
+  ModalAccessibleContextType,
+} from "./ModalAccessibleContext"
 
 import { warn } from "@kaizen/component-library/util/console"
-import { ID_DESCRIBEDBY, ID_LABELLEDBY } from "./constants"
 
 const styles = require("./GenericModal.scss")
 
-interface Props {
+interface ContainerProps {
   readonly isOpen: boolean
   readonly children: React.ReactNode
   readonly focusLockDisabled?: boolean
   readonly onEscapeKeyup?: (event: KeyboardEvent) => void
   readonly onOutsideModalClick?: (event: React.MouseEvent) => void
 }
 
+interface Props extends ContainerProps, ModalAccessibleContextType {}
```

Comment:
/nitpick I couldn't grok what this was for on a first parse. Could you rename to something like `GenericModalProps`?

Reply:
Ah, yep! Can do.

Reply:
That’s updated now, @ActuallyACat. Mind ✅ ing it again?

Reply:
@ActuallyACat @makenosound Thanks for sharing. That all makes sense ;-) 

The only other thing worth noting is the colour of the Primary button on these models needs to be upgraded to the Cluny 500 Primary Action.

Reply:
Thanks, @katsambarcus! I _think_ there’s a separate WIP branch to update those buttons everywhere, but perhaps @ActuallyACat can confirm.

If we need to do some work here I’m happy to adjust all the modals while I’m in this zone.